build/Helix/CopyVisualTreeVerificationFiles.ps1

```diff
@@ -0,0 +1,2 @@
+$picturesFolder = [Environment]::GetFolderPath('MyPictures')
+Copy-Item $picturesFolder\*.xml $Env:HELIX_WORKITEM_UPLOAD_ROOT
```

Comment:
We probably don't need this file at all. Visual Tree Verification was something I removed from the steps as I don't think we have tests like that.